openreview/journal/invitation.py

```diff
@@ -4817,6 +4811,13 @@ def set_camera_ready_verification_invitation(self):
             'dateprocesses': [self.ae_reminder_process]
         }
 
+        if self.journal.has_publication_chairs():
```

Comment:
Does this mean if the journal has publication chairs, then they submit the camera ready verification instead of AEs?

Reply:
don't merge it, I will fix this.

Reply:
fixed